next-tavla/app/(admin)/components/Footer/styles.module.css

```diff
@@ -10,3 +10,15 @@
         padding: 4rem 2rem 2rem;
     }
 }
+
+.flexDirection {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+@media (max-width: 550px) {
+    .flexDirection {
+        flex-direction: column;
+    }
+}
```

Comment:
syntes klassenavnet er litt uklart,ville kanskje kalt den for meta eller noe sånt